### PR TITLE
Change phenotypic profiles method to work with the new monarch schema

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

**Comment by balhoff** [comment]
@Shalsh23 can you please create a working branch on GitHub for working with the new pipeline, and open this pull request against than branch instead of `master`?

File: src/main/scala/org/phenoscape/kb/EQForGene.scala [hunk 1]
```diff
@@ -56,9 +57,9 @@ object EQForGene {
   }
 
   def annotationsQuery(geneIRI: IRI): Query = {
-    select_distinct('annotation) from "http://kb.phenoscape.org/" where bgp(
-      t('annotation, rdfType, Vocab.AnnotatedPhenotype),
-      t('annotation, Vocab.associated_with_gene, geneIRI))
+    select_distinct('association) from "http://kb.phenoscape.org/" where bgp(
+      t('association, rdfType, association),
+      t('association, associationHasSubject, geneIRI))
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/kb/EQForGene.scala]: For future-proofing, please check the `association` predicate here (has_phenotype).

File: src/main/scala/org/phenoscape/kb/Gene.scala [hunk 2]
```diff
@@ -161,16 +161,17 @@ object Gene {
...
-        t('pheno_instance, rdfType, 'annotation)),
-      optional(bgp(t('pheno_instance, dcSource, 'source))),
-      new ElementFilter(new E_NotOneOf(new ExprVar('annotation), new ExprList(List[Expr](
-        new NodeValueNode(AnnotatedPhenotype),
+        t('association, rdfType, association),
+        t('association, associationHasSubject, iri),
+        t('association, associationHasObject, 'phenotype),
+        t('association, associationHasPredicate, has_phenotype)),
+      optional(bgp(t('association, dcSource, 'source))),
+      new ElementFilter(new E_NotOneOf(new ExprVar('phenotype), new ExprList(List[Expr](
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/kb/Gene.scala]: Now I think you can take this filter out completely.

File: src/main/scala/org/phenoscape/kb/GenePhenotypeAnnotation.scala [hunk 3]
```diff
@@ -68,19 +68,21 @@ object GenePhenotypeAnnotation {
...
-          t('annotation, associated_with_gene, 'gene) ::
+          t('association, rdfType, association) ::
+          t('association, associationHasPredicate, has_phenotype) ::
+          t('association, associationHasSubject, 'gene) ::
+          t('association, associationHasObject, 'phenotype) ::
           t('gene, rdfsLabel, 'gene_label) ::
-          t('annotation, rdfType, 'phenotype) ::
           t('phenotype, rdfsLabel, 'phenotype_label) ::
-          t('annotation, associated_with_taxon, 'taxon) ::
+          t('association, in_taxon, 'taxon) ::
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/kb/GenePhenotypeAnnotation.scala]: `in_taxon` is a property of the gene, not the association.

File: src/main/scala/org/phenoscape/kb/GenePhenotypeAnnotation.scala [hunk 4]
```diff
@@ -68,19 +68,21 @@ object GenePhenotypeAnnotation {
...
           phenotypeTriple ++
             taxonPatterns: _*),
-      optional(bgp(t('annotation, dcSource, 'source))),
+      optional(bgp(t('association, dcSource, 'source))),
       new ElementFilter(new E_NotOneOf(new ExprVar('phenotype), new ExprList(List[Expr](
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/kb/GenePhenotypeAnnotation.scala]: Now I think you can take this filter out completely.

File: src/main/scala/org/phenoscape/kb/KBVocab.scala [hunk 5]
```diff
@@ -46,11 +46,14 @@ object KBVocab {
   val serially_homologous_to = ObjectProperty("http://purl.obolibrary.org/obo/RO_HOM0000027")
 
   val oban = "http://purl.org/oban/"
+  val association = IRI.create("http://purl.org/oban/association")
```
Comment (balhoff) [inline src/main/scala/org/phenoscape/kb/KBVocab.scala]: Might as well reuse the `oban` prefix like in the lines below.

**Comment by Shalsh23** [comment]
@balhoff Addressed these requests for changes for now. But they will need to be further tweaked as we refine our understanding of the new data model?

**Comment by balhoff** [comment]
Yep, let's leave open for now until we get the data a little better sorted out.

**Pull request merged**